samplesheet_nobc_nodx.csv

```diff
@@ -1,3 +1,7 @@
-sample,fastq,barcode,genome,transcriptome
-K562_RUN1_REP3,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode01.fastq.gz,,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/reference/hg19_KCMF1.fa,
-HEPG2_RUN3_REP5,https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/fastq/demultiplexed/barcode02.fastq.gz,,,
+sample,sample_path,barcode,genome,transcriptome,condition
```

Comment:
```suggestion
sample,input_file,barcode,genome,transcriptome,condition
```

Reply:
Can we use `input_file` here instead @yuukiiwa ? More generically applicable.

Reply:
Thanks for the reminder! I have fixed them in all samplesheets.